apps/docs-v2/components/common/CommonHeader.tsx

```diff
@@ -80,7 +80,12 @@ export const CommonHeaderTitle = () => {
       {hasImage ? (
         <Breadcrumbs>
           {breadcrumbs.map((title, index) => {
-            const routeTo = `/${breadcrumbs.slice(0, index + 1).join("/").toLowerCase()}`;
+            let routeTo
+            if (title === 'Foundation'){
+              routeTo = '/foundation/overview'
```

Comment:
Please don't hard code. Find its parent and render the url as `/[parent]/[first_child]`

Reply:
else statement will take it mission, this line will redirect to overview of foundation page instead of index file cuz we dont have it